### PR TITLE
Description updates

### PR DESCRIPTION
**Pull request by davmayd**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/sios-datakeeper-main.template.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Description: >-
-  This template deploys two Windows Server Failover Clustering (WSFC) nodes with SQL
+  This template deploys two or three Windows Server Failover Clustering (WSFC) nodes with SQL
   Server 2014 and DataKeeper replication software. This template also deploys the
```
Comment (MarciaRieferJohnston) [inline templates/sios-datakeeper-main.template.yaml:4]: @davmayd Do we need to keep a year on "Server 2014"? I'd delete the year if we can for future-proofing.

File: templates/sios-datakeeper-main.template.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-  required AD architecture outlined in "Implementing Active Directory Domain Services
-  in the AWS Cloud" **WARNING** This template creates Amazon EC2 Windows instance
+  required Active Directory architecture as deployed by the Active Directory Domain Services
+  on the AWS Cloud Quick Start. **WARNING** This template creates Amazon EC2 Windows instance
```
Comment (MarciaRieferJohnston) [inline templates/sios-datakeeper-main.template.yaml:6]:
```suggestion
  on AWS Quick Start. **WARNING** This template creates Amazon EC2 Windows instance
```

File: templates/sios-datakeeper.template.yaml [hunk 3]
```diff
@@ -1,12 +1,11 @@
 ---
 AWSTemplateFormatVersion: '2010-09-09'
-Description: 'This template deploys a two or three node Windows Server Failover Cluster (WSFC)
+Description: This template deploys a two or three node Windows Server Failover Cluster (WSFC)
   with SQL Server 2017 and SIOS DataKeeper replication software. This template is
-  intended to be installed into an existing VPC that was built using the sample reference
-  architecture titled: "Implementing Active Directory Domain Services in the AWS Cloud"
-  **WARNING** This template creates Amazon EC2 Windows instance and related resources.
+  intended to be installed into an existing VPC that was built using the Active Directory Domain Services
+  on the AWS Cloud Quick Start **WARNING** This template creates Amazon EC2 Windows instance and related resources.
```
Comment (MarciaRieferJohnston) [inline templates/sios-datakeeper.template.yaml:6]:
```suggestion
  on AWS Quick Start. **WARNING** This template creates Amazon EC2 Windows instance and related resources.
```

File: templates/sios-datakeeper.template.yaml [hunk 4]
```diff
@@ -1,12 +1,11 @@
 ---
 AWSTemplateFormatVersion: '2010-09-09'
-Description: 'This template deploys a two or three node Windows Server Failover Cluster (WSFC)
+Description: This template deploys a two or three node Windows Server Failover Cluster (WSFC)
```
Comment (MarciaRieferJohnston) [inline templates/sios-datakeeper.template.yaml:3]:
```suggestion
Description: This template deploys a two- or three-node Windows Server Failover Cluster (WSFC)
```

File: templates/sios-datakeeper.template.yaml [hunk 5]
```diff
@@ -1,12 +1,11 @@
 ---
 AWSTemplateFormatVersion: '2010-09-09'
-Description: 'This template deploys a two or three node Windows Server Failover Cluster (WSFC)
+Description: This template deploys a two- or three-node Windows Server Failover Cluster (WSFC)
   with SQL Server 2017 and SIOS DataKeeper replication software. This template is
```
Comment (MarciaRieferJohnston) [inline templates/sios-datakeeper.template.yaml:4]: @davmayd Same question as above (re: "Server 2017" here).

**Review by MarciaRieferJohnston — COMMENTED** [review]
Looks good. See my questions about the years after "SQL Server."

File: templates/sios-datakeeper.template.yaml (hunk 5 above)
Comment (davmayd) [reply templates/sios-datakeeper.template.yaml:4]: @MarciaRieferJohnston it defaults to 2017 but allows other versions (https://github.com/aws-quickstart/quickstart-sios-datakeeper/blob/main/templates/sios-datakeeper.template.yaml#L384), if you'd prefer to omit the SQL server version I have no objection

File: templates/sios-datakeeper-main.template.yaml (hunk 1 above)
Comment (MarciaRieferJohnston) [inline templates/sios-datakeeper-main.template.yaml:4]:
```suggestion
  Server and DataKeeper replication software. This template also deploys the
```

File: templates/sios-datakeeper.template.yaml (hunk 5 above)
Comment (MarciaRieferJohnston) [reply templates/sios-datakeeper.template.yaml]: Thanks, Dave. Done.

**Review by MarciaRieferJohnston — APPROVED** [review]

**Pull request merged**